lib/ex_webrtc/rtp_transceiver.ex

```diff
@@ -3,6 +3,8 @@ defmodule ExWebRTC.RTPTransceiver do
   Implementation of the [RTCRtpTransceiver](https://www.w3.org/TR/webrtc/#dom-rtcrtptransceiver).
   """
 
+  require Logger
```

Comment:
Dunno how you all feel about Loggers inside libraries, let me know if you'd like a different approach but this warning is useful for implementing devs.

Reply:
We put on warning things that are totally unexpected and the rest of things that are correct on debug so I would put the new log on debug :wink: